package.json

```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "webpack": "babel-node ./node_modules/webpack/bin/webpack",
-    "start": "babel-node ./node_modules/webpack-dev-server/bin/webpack-dev-server --open",
+    "build": "NODE_ENV=production babel-node ./node_modules/webpack/bin/webpack --config webpack/prod.config.js",
+    "dev": "NODE_ENV=development babel-node ./node_modules/webpack-dev-server/bin/webpack-dev-server --config webpack/dev.config.js",
+    "start": "npm run build && babel-node  server.js",
```

Comment:
Keep the previous start script, so we could use webpack-dev-server for development.
And use `start:prod` for the production build.